pkgs/applications/networking/instant-messengers/pidgin-plugins/pidgin-indicator/default.nix

```
@@ -0,0 +1,33 @@
+{ autoreconfHook
+, fetchFromGitHub
+, glib
+, intltool
+, lib
+, libappindicator-gtk2
+, libtool
+, pidgin
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pidgin-indicator";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "philipl";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-CdA/aUu+CmCRbVBKpJGydicqFQa/rEsLWS3MBKlH2/M=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ glib intltool libappindicator-gtk2 libtool pidgin ];
+
+  meta = {
+    description = "A AppIndicator/KStatusNotifierItem Plugin for Pidgin";
+    homepage = "https://github.com/philipl/pidgin-indicator";
+    maintainers = with lib.maintainers; [ imalison ];
+    license = lib.licenses.gpl2;
```

```suggestion
    license = licenses.gpl2;
```
gpl2Plus or gpl2Only?

---

I'm not sure how to tell. I just saw gpl2 here:

https://github.com/philipl/pidgin-indicator/blob/master/COPYING